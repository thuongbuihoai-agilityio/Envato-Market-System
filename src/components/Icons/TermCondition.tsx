export const TermCondition = () => (
  <svg
    width="24"
    height="24"
    viewBox="0 0 24 24"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9 14L10.7528 15.4023C11.1707 15.7366 11.7777 15.6826 12.1301 15.2799L15 12M16 4H17C19.2091 4 21 5.79086 21 8V18C21 20.2091 19.2091 22 17 22H7C4.79086 22 3 20.2091 3 18V8C3 5.79086 4.79086 4 7 4H8M16 4C16 5.10457 15.1046 6 14 6H10C8.89543 6 8 5.10457 8 4M16 4C16 2.89543 15.1046 2 14 2H10C8.89543 2 8 2.89543 8 4"
      strokeWidth="1.5"
      stroke="currentColor"
      strokeLinecap="round"
    ></path>
  </svg>
);
