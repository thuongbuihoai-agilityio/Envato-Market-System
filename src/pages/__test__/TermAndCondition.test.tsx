import { render } from '@testing-library/react';

// Pages
import TermCondition from '@app/pages/Setting/TermAndCondition';

describe('Term and Condition Component', () => {
  it('Should render match with snapshot.', async () => {
    const { container } = render(<TermCondition />);

    expect(container).toMatchSnapshot();
  });
});
