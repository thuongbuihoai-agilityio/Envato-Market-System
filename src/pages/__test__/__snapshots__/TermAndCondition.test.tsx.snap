// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Term and Condition Component Should render match with snapshot. 1`] = `
<div>
  <div
    class="css-0"
  >
    <h3
      class="chakra-text css-2sbtmu"
    >
      Term and Conditions
    </h3>
    <div
      class="css-19idom"
    >
      <h4
        class="chakra-text css-1gmhtzj"
      >
        1. Definitions
      </h4>
      <p
        class="chakra-text css-1wtnvd2"
      >
        What you do own when you buy an are the keys to a non-fungible – perhaps unique – token. That token is yours to trade or hold or display in Decentraland. But the digital file associated with an NFT is just as easy to copy and paste and download as any other – the Finally, players lose their NFTs sometimes according to the rules and regulations of the NFT game.
      </p>
      <p
        class="chakra-text css-b0xv85"
      />
    </div>
    <div
      class="css-19idom"
    >
      <h4
        class="chakra-text css-1gmhtzj"
      >
        2. Acceptance
      </h4>
      <p
        class="chakra-text css-1wtnvd2"
      >
        Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.Capacity. You confirm that you have the legal capacity to receive and hold and make use of the NFT under French jurisdiction and any other relevant jurisdiction.Acceptance. By participating in the Sale, You accept and agree to these Terms and Conditions without any condition or restriction. If You do not agree to this Contract, You shall not participate in the Sale made by the Company Exercitation veniam consequat sunt nostrud amet.Capacity. You confirm that you have the legal capacity to receive and hold find to end.Contract, You shall not participate in the Sale made by the Company Exercitation venia
      </p>
      <p
        class="chakra-text css-b0xv85"
      />
      <div
        class="css-ze22id"
      >
        <p
          class="chakra-text css-1vr8d7m"
        >
          In reality, the most important aspect of a great dashboard is the part that gets the least amount of attention: The underlying data. More than any other aspect, the data will make or break a dashboard.Within this definition, successful administration appears to rest on three basic skills, which we will call technical, and conceptual. 
        </p>
      </div>
    </div>
    <div
      class="css-19idom"
    >
      <h4
        class="chakra-text css-1gmhtzj"
      >
        3. The Sale
      </h4>
      <p
        class="chakra-text css-1wtnvd2"
      >
        The Company offers featuring the Betonyou universe. The holders of one or more NFTs will be able to win cryptos while playing video games. In the future, the Company plans to develop its own games and Metaverse around the Betonyou universe (“Project”).
      </p>
      <p
        class="chakra-text css-b0xv85"
      >
        To release the NFTs and fund the project, the Company offers NFTs from a dedicated website("Sale"). The web address of this website will be given at the time of the mint. The NFT acquisition does not confer any rights on the Company or in the future development.
      </p>
    </div>
    <div
      class="css-19idom"
    >
      <h4
        class="chakra-text css-1gmhtzj"
      >
        4. Purchaser’s obligations
      </h4>
      <p
        class="chakra-text css-1wtnvd2"
      >
        To the fullest extent permitted by applicable law, You undertake to indemnify, defend and hold harmless the Company from and against all claims, demands, actions, damages, losses, costs and expenses (including attorneys’ fees) that arise from or relate to (i) your Subscription or use of the NFTs; (ii) your responsibilities or obligations under this Contract; and (iii) your breach of this Contract.
      </p>
      <p
        class="chakra-text css-b0xv85"
      >
        Company undertakes to act with the care normally expected from a professional in his field and to comply with the best practice in force. The best endeavor obligation only binds the Company.
      </p>
    </div>
  </div>
</div>
`;
